cadc-vos-server/src/main/java/ca/nrc/cadc/vos/server/transfers/TransferRunner.java

```diff
@@ -386,17 +387,34 @@ private void doTransferRedirect(Transfer trans, List<Parameter> additionalParame
 
             // standard redirect
             StringBuilder sb = new StringBuilder();
-            sb.append("/").append(job.getID()).append("/results/transferDetails");
+            sb.append("/");
```

Comment:
I guess you can append the jobID right here as well since they both have it.

Reply:
done